data/virt_autotest/tw_ipxe/menu.ipxe

```diff
@@ -1,28 +1,38 @@
 # This is the ipxe menu for installing TW on O3
 # path: user@ariel:/srv/tftpboot/ipxe/menu.ipxe
 # ---------- file begin ----------
-
 #!ipxe
 
-dhcp
+# dhcp should be already done by boot script
+#dhcp
 
 #set variables
 set menu-timeout 30000
+set server-name netboot
+iseq ${next-server} 10.150.1.11   && set server-name openqa.opensuse.org ||
+iseq ${next-server} 10.168.192.10 && set server-name qe.nue2.suse.org ||
+iseq ${next-server} 10.162.0.1    && set server-name qanet.qa.suse.de ||
```

Comment:
Would you please give more background information why three  server-name are needed? Related with NUE1 lab move?

Reply:
menu.ipxe in O3 was used and controled by us on O3 only, so we tracked in git under our test directory.

However days ago, it was adapted to support both PXE and iPXE boot by tools team, and apply it on both O3 and OSD. ie. it began to be managed by tools team.

We still has the access to this file and edited it on the server directly. It's better that it is still be tracked in git to avoid accedental breakage.

Reply:
Thankyou Julie for mentioning it

Reply:
okay, thanks for the info